app/views/admin/bookmarks/index.html.haml

```diff
@@ -0,0 +1,77 @@
+- content_for :page_title do
+  Bookmarks
+
+- content_for :menu do
+  .item.hidden{ data: { behavior: "bulk-edit-menu" } }
+    .divider
+    .header Bulk Actions
+    .menu
+      .item
+        .ui.fluid.icon.button
+          %i.download.icon
+          recache all
+
+      .item
+        .ui.fluid.icon.negative.basic.button
+          %i.trash.icon
+          delete all
+
+
+- content_for :body do
```

Comment:
`content_for :body` seems redundant. Why not just have this part of the template be at the top level?

Reply:
Good question. At this rate I'm just copying this pattern from 7 previous projects :P

Reply:
![](https://media.giphy.com/media/cg5FwpvDmhIcM/giphy.gif)